exercises/practice/leap/.meta/config.json

```diff
@@ -19,5 +19,5 @@
     ]
   },
   "source": "JavaRanch Cattle Drive, exercise 3",
-  "source_url": "http://www.javaranch.com/leap.jsp"
+  "source_url": "https://www.javaranch.com/leap.jsp"
```

Comment:
It redirects to this page: `https://coderanch.com/t/718816/Leap`. So might as well switch them